src/java.base/share/classes/java/util/Locale.java

```diff
@@ -230,40 +230,26 @@
  * implementations in a Java Runtime Environment might not support any
  * particular Unicode locale attributes or key/type pairs.
  *
- * <h3>Creating a Locale</h3>
+ * <h3><a id="ObtainingLocale">Obtaining a Locale</a></h3>
  *
- * <p>There are several different ways to create a {@code Locale}
+ * <p>There are several ways to obtain a {@code Locale}
  * object.
  *
  * <h4>Builder</h4>
  *
  * <p>Using {@link Builder} you can construct a {@code Locale} object
  * that conforms to BCP 47 syntax.
  *
- * <h4>Constructors</h4>
+ * <h4>Factory Method</h4>
  *
- * <p>The {@code Locale} class provides three constructors:
- * <blockquote>
- * <pre>
- *     {@link #Locale(String language)}
- *     {@link #Locale(String language, String country)}
- *     {@link #Locale(String language, String country, String variant)}
- * </pre>
- * </blockquote>
- * These constructors allow you to create a {@code Locale} object
- * with language, country and variant, but you cannot specify
- * script or extensions.
- *
- * <h4>Factory Methods</h4>
- *
- * <p>The method {@link #forLanguageTag} creates a {@code Locale}
+ * <p>The method {@link #forLanguageTag} obtains a {@code Locale}
  * object for a well-formed BCP 47 language tag.
  *
  * <h4>Locale Constants</h4>
  *
  * <p>The {@code Locale} class provides a number of convenient constants
- * that you can use to create {@code Locale} objects for commonly used
- * locales. For example, the following creates a {@code Locale} object
+ * that you can use to obtain {@code Locale} objects for commonly used
+ * locales. For example, the following obtains a {@code Locale} object
```

Comment:
I'd replace this part (and the example below) with something like,

    For example, {@code Locale.US} is the {@code Locale} object for the United States.